server/routes/html.coffee

```diff
@@ -0,0 +1,198 @@
+User = require '../users/User'
+UserHandler = require '../users/user_handler'
+LevelSession = require '../levels/sessions/LevelSession'
+Campaign = require '../campaigns/Campaign'
+Level = require '../levels/Level'
+config = require '../../server_config'
+log = require 'winston'
+Mandate = require '../models/Mandate'
+utils = require '../lib/utils'
+cheerio = require 'cheerio'
+en = require '../../app/locale/en'
+_ = require 'lodash'
+
+translate = (key) ->
+  html = /^\[html\]/.test(key)
+  key = key.substring(6) if html
+
+  t = en.translation
+  #TODO: Replace with _.property when we get modern lodash
+  path = key.split(/[.]/)
+  while path.length > 0
+    k = path.shift()
+    t = t[k]
+    return key unless t?
+
+  return out =
+    text: t
+    html: html
+
+
+makeConext = (req) ->
+  view =
+    isIPadBrowser:  -> false
+    isMobile:  -> false
+    isOldBrowser: -> false
+    forumLink: -> 'http://discourse.codecombat.com/'
+    showAds: -> true
+
+  me = 
+    getPhotoURL:  -> ''
+    isAnonymous:  -> true
+    get: (what) -> ''
+    level:  -> ''
+    displayName:  -> ''
+    isTeacher:  -> false
+    isOnPremiumServer: -> false
+    isAdmin: -> false
+    gems: -> 0
+    isPremium: -> false
```

Comment:
Since user is on the req object, could provide that instead to make the page more accurate. Would want to make the mongoose User have the same functions as the Backbone User, though...